CONTRIBUTING.md

```diff
@@ -92,3 +162,42 @@ We welcome contributions, large or small, including:
 If you would like to contribute new APIs, please check the [CODEOWNERS](https://github.com/iTwin/itwinjs-core/blob/master/.github/CODEOWNERS) file to ensure your contributions align with the package owner's and project goals.
```

Comment:
I realize that this is from before, but I don't understand what this sentence is trying to say. What can a contributor possibly learn from looking at the CODEOWNERS file?

Reply:
okay, modified